src/scripts.js

```diff
@@ -150,13 +151,88 @@ let stepsTrendingButton = document.querySelector('.steps-trending-button');
+let flightsOfStairsInput = document.querySelector(".stairs-input");
+let activitySubmitButton = document.querySelector(".activity-submit-button");
+let showSleepFormButton = document.querySelector(".show-sleep-form");
+let showActivityFormButton = document.querySelector(".show-activity-form");
+let showHydrationFormButton = document.querySelector(".show-hydration-form");
+let sleepInputForm = document.querySelector(".sleep-input-form");
+let activityInputForm = document.querySelector(".activity-input-form");
+let hydrationInputForm = document.querySelector(".hydration-input-form");
+let successfulSubmit = document.querySelector(".successful-submit");
+let failedSubmit = document.querySelector(".failed-submit");
+let clearButton = document.querySelector(".clear-button");
+let errorMessage = document.querySelector(".error-message");
 
 mainPage.addEventListener('click', showInfo);
 profileButton.addEventListener('click', showDropdown);
 stairsTrendingButton.addEventListener('click', updateTrendingStairsDays());
 stepsTrendingButton.addEventListener('click', updateTrendingStepDays());
 
+submitButton.addEventListener('click', postSleepHelper );
+hydrationSubmitButton.addEventListener('click', postHydrationHelper);
+activitySubmitButton.addEventListener('click', postActivityHelper);
+showSleepFormButton.addEventListener('click', showSleepForm);
+showActivityFormButton.addEventListener('click', showActivityForm);
+showHydrationFormButton.addEventListener('click', showHydrationForm);
+clearButton.addEventListener('click', hideForms)
+
+function showErrorMessage() {
+  errorMessage.classList.remove("hide")
+}
+
+function hideForms() {
+  sleepInputForm.classList.add("hide");
+  activityInputForm.classList.add("hide");
+  hydrationInputForm.classList.add("hide");
+  successfulSubmit.classList.add("hide");
+  failedSubmit.classList.add("hide");
+  clearButton.classList.add("hide");
+  errorMessage.classList.add("hide")
+}
+
+function showSleepForm() {
+  sleepInputForm.classList.remove("hide");
+  activityInputForm.classList.add("hide");
+  hydrationInputForm.classList.add("hide");
+  successfulSubmit.classList.add("hide");
+  failedSubmit.classList.add("hide");
+  clearButton.classList.remove("hide");
+  errorMessage.classList.add("hide")
+}
+
+function showActivityForm() {
+  activityInputForm.classList.remove("hide");
+  sleepInputForm.classList.add("hide");
+  hydrationInputForm.classList.add("hide");
+  successfulSubmit.classList.add("hide");
+  failedSubmit.classList.add("hide");
+  clearButton.classList.remove("hide");
+  errorMessage.classList.add("hide")
+}
+
+function showHydrationForm() {
+  hydrationInputForm.classList.remove("hide");
+  activityInputForm.classList.add("hide");
+  sleepInputForm.classList.add("hide");
+  successfulSubmit.classList.add("hide");
+  failedSubmit.classList.add("hide");
+  clearButton.classList.remove("hide");
+  errorMessage.classList.add("hide")
```

Comment:
probably dry this up too and create a function maybe that can hide

Reply:
Script refactor!